webapp/src/main/java/life/qbic/security/AuthenticatedUser.java

```diff
@@ -16,17 +16,18 @@
 @Component
 public class AuthenticatedUser {
 
-    @Autowired
-    private UserRepository userRepository;
+    //@Autowired
+   // private UserRepository userRepository;
 
     private Optional<Authentication> getAuthentication() {
         SecurityContext context = SecurityContextHolder.getContext();
         return Optional.ofNullable(context.getAuthentication())
                 .filter(authentication -> !(authentication instanceof AnonymousAuthenticationToken));
     }
 
-    public Optional<TestUser> get() {
-        return getAuthentication().map(authentication -> userRepository.findByUsername(authentication.getName()));
+    public Optional<User> get() {
+        //return getAuthentication().map(authentication -> userRepository.findByUsername(authentication.getName()));
+        return null; //todo
```

Comment:
Can you add a more detailed TODO statement here please, so we know what needs to be done